pkg/providers/aws/provider_redis.go

```diff
@@ -158,6 +159,66 @@ func (p *AWSRedisProvider) createElasticacheCluster(ctx context.Context, r *v1al
 		return nil, "modify elasticache cluster in progress", nil
 	}
 
+	// Add Tags to AWS Elasticache
+	if err != nil {
+		logrus.Errorf("Error Failed to get strategy config: %s", err)
+	}
+	cacheSvcTag := elasticache.New(session.Must(session.NewSession(&aws.Config{Region: aws.String(stratCfg.Region)})))
```

Comment:
Same as others, do we need to create a new session for the tagging?

Reply:
same as above